index.ipynb

```diff
@@ -19,6 +19,10 @@
     "\n",
     "[![orbit](docs/source/examples/perihelion_advance.png)](docs/source/examples/Visualizing_advance_of_perihelion_of_a_test_particle_in_Schwarzschild_space-time.ipynb)\n",
     "\n",
+    "## [Simulating orbit of a test particle in Schwarzschild space-time](docs/source/examples/Simulating_orbit_of_a_test_particle_in_Schwarzschild_space-time.ipynb)\n",
```

Comment:
Title of the notebook will be: Animations in EInsteinPy

Reply:
Done